src/trackedit/itrackeditinteraction.h

```diff
@@ -56,6 +56,11 @@ class ITrackeditInteraction : MODULE_EXPORT_INTERFACE
     virtual void newLabelTrack() = 0;
     virtual void deleteTracks(const TrackIdList& trackIds) = 0;
     virtual void duplicateTracks(const TrackIdList& trackIds) = 0;
+    virtual void moveTracksUp(const TrackIdList& trackIds) = 0;
```

Comment:
or add only two methods like
```
virtual void moveTracks(const TrackIdList& trackIds, TrackMoveDir dir) =0;
virtual void moveTracks(const TrackIdList& trackIds, int pos) =0;
```

Reply:
I'm afraid that if someone else from the community decides to add a new type, they will do it without switching to a parameter and we cannot require them to refactor, this is roughly how open source works